godot-macros/src/class/derive_godot_class.rs

```diff
@@ -206,6 +197,43 @@ pub fn derive_godot_class(item: venial::Item) -> ParseResult<TokenStream> {
     })
 }
 
+fn make_with_base_impl(base_field: &Option<Field>, class_name: &Ident) -> TokenStream {
+    if let Some(Field { name, ty, .. }) = base_field {
+        // Apply the span of the field's type so that errors show up on the field's type.
```

Comment:
Did you mean "on the field"?
```suggestion
        // Apply the span of the field's type so that errors show up on the field.
```